http_handlers.go

```diff
@@ -50,23 +61,43 @@ func webhook(c *gin.Context) {
 			generatorUrl, err := url.Parse(alert.GeneratorURL)
 			if err != nil {
 				err = errors.Wrap(err, "Could not get generator url")
-				_ = bugsnag.Notify(err)
+				_ = bugsnag.Notify(err, ctx,
+					bugsnag.MetaData{
+						"Alert": {
+							"Name": alertName,
+							"GeneratorURL": alert.GeneratorURL,
+						},
+					})
 				clog.Error(err.Error())
 			}
 
 			// from the chart url get the expressions to build the charts
 			generatorQuery, err := url.ParseQuery(generatorUrl.RawQuery)
 			if err != nil {
 				err = errors.Wrap(err, "Could not get query from generator url")
```

Comment:
And for consistency perhaps:
```suggestion
				err = errors.Wrap(err, "Could not parse query from generator url")
```